docs/overview/directory-structure.md

```diff
@@ -21,7 +21,11 @@ paths will be familiar to Django developers.
   [Django models](https://docs.djangoproject.com/en/1.8/topics/db/models/)
   file.  Defines Zulip's database tables.
 
-* `zerver/lib/actions.py` Most code doing writes to user-facing database tables.
+* `zerver/lib/*.py` Most library code.
+
+* `zerver/lib/actions.py` Much of our code that writes to user-facing
+   database tables lives here, for legacy reasons, but we have moved
+   some of its former responsibilities to other library files.
```

Comment:
I think the most accurate description of "actions" is that it contains essentially all database writes that results in server=>client pushes (in particular, all `send_event` server=>client push calls are from `actions.py`, and that will likely be a policy we keep even if we move it to a directory)